client/blocks/conversations/list.jsx

```diff
@@ -169,7 +169,7 @@ export class ConversationCommentList extends React.Component {
 		const { commentIds, expansions, commentsTree, sortedComments } = this.props;
 
 		const minId = min( commentIds );
-		const startingCommentIds = sortedComments
+		const startingCommentIds = ( sortedComments || [] )
```

Comment:
Would this be better in `connect` like [this](https://github.com/Automattic/wp-calypso/pull/18872/files#diff-27bdbf2360c213094ee0784e459c5e29L303)?

Reply:
thats a good thought.  I'm not sure where it'll be most natural to put the null--> empty list conversion once this component cares about displaying placeholders.

when making placeholders i'll revisit the `|| []` placement